features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/assets/api/themes/store/js/console.js

```diff
@@ -0,0 +1,59 @@
+(function () {
+
+    var WARN_MSG = '[WARN]';
+    var INFO_MSG = '[INFO]';
+    var ERR_MSG = '[ERR]';
+
+    var main = function () {
+
+        if (checkConsoleObject()) {
+            init(console);
+            return;
+        }
+
+        initEmptyLogger(console);
+    };
+
+    var checkConsoleObject = function () {
+        if (typeof console==undefined) {
+            return false;
+        }
+        return true;
+    };
+
+    var init = function (console) {
+
+        console.info = function (msg) {
+            console.log(INFO_MSG + msg);
+        };
+
+        console.warn = function (msg) {
+            console.log(WARN_MSG + msg);
+        };
+
+        console.err = function (msg) {
+            console.log(ERR_MSG + msg);
+        };
+    }
+
+    var initEmptyLogger = function (console) {
+        console={};
+        console.info = function (msg) {
+
```

Comment:
This script has been added as a shim in order to console log to fail gracefully.